src/components/tutorials/basic-tutorial.js

```diff
@@ -14,23 +14,23 @@ class BasicTutorial extends React.Component {
+            <p>Since the API and scoring logic are updated on a regular basis as per the annual publication of the QPP Final Rule, please note that some numbers (especially scoring) may be inaccurate. The <a href='https://qpp-submissions-sandbox.navapbc.com/'>Interactive Docs</a> and Developer Preview will always reflect the latest design.</p>
+            <p>Let's walk through an example of how you might submit performance data as a registry API user!</p>
+            <h2 className='ds-h2' id='creating-a-measurement-set'>
               <a
                 className='tutorial-header-link'
-                href='#creating-a-submission'>
-                Creating a new submission
+                href='#creating-a-measurement-set'>
+                Create a new measurement set
             </a>
             </h2>
-            <p>We need to create a <em>submission</em> first. We can do that by asking the API to create a submission record in the CMS database. In API terms, this means making a <code>POST</code> (synonym for <em>create</em>) request to the <code>/submissions</code> endpoint. </p>
+            <p>We need to create a <em>measurement set</em> first. We can do that by asking the API to create a mesurement set record in the CMS database. In API terms, this means making a <code>POST</code> (synonym for <em>create</em>) request to the <code>/measurement-sets</code> endpoint. </p>
             <p>When submitting, you can pass an <code>Accept</code> header to specify the API version and type of response back by using our custom mime type, <code>application/vnd.qpp.cms.gov.v1+json</code>. We support JSON and XML by using <code>+json</code> or <code>+xml</code>. Right now, only version <code>v1</code> is supported. You can also continue to use a standard <code>application/json</code> or <code>application/xml</code> header, which will point to the latest version (currently v1).</p>
-            <p>We'll also need to supply some information to tell CMS how to identify this particular submission, which you can see below - every submission is unique to the combination of the fields provided. Note that we enforce fake TINs (starting with <code>000</code>) during the preview period to avoid accidentally collecting personally identifiable information. We can submit performance data in this first request, but we'll do it in a future request to keep this one small. Take a look then click response!</p>
+            <p>We'll also need to supply some information to tell CMS how to identify this particular submission, which you can see below - every submission is unique to the combination of the fields provided. Note that we enforce fake TINs (starting with <code>000</code>) in the Developer Preview to avoid accidentally collecting personally identifiable information. We can submit performance data in this first request, but we'll do it in a future request to keep this one small. Take a look then click response!</p>
             <InlineApiExample
```

Comment:
In the table above the request payload, TIN is 000456789 and NPI is 0123456789. Update the TIN and NPI in the request payload to match.